cylc/flow/network/schema.py

```diff
@@ -753,8 +759,8 @@ class Meta:
             update relates to a workflow reload.
         '''),
     )
-    run_mode = String(
-        description="The scheduler's run-mode e.g. `live`.",
+    run_mode = WorkflowRunMode(
+        description="The scheduler's run-mode e.g. `Live`.",
```

Comment:
can we avoid changing case? It will make it harder to make the ui work with both 8.4.x and 8.5.x.

Reply:
Not sure what Tim means by "changing case", but this change seems to be problematic for a `workflows` query

```gql
{
  workflows {
    id
    runMode
  }
}
```

> Enum 'WorkflowRunMode' cannot represent value: ''

Reply:
It's not a simple case change.. It's a type change from enum value to enum, it's part of the graphql changes about enums..
If you want the lowercase then we'd have to not use enums, or some other workaround.

Reply:
It means that I don't want to change `live` -> `Live`. Case as in upper case and lower case.

Reply:
The problem is `live` is not an enum, but `Live' is.

Reply:
But can `live` not be an enum if you change line 631?

Reply:
@wxtim The case of the Enum member name is correct (`Live`). This is only relevant to the GraphQL API; the only way users will see this is in a help icon in the UI where the options are all sentence-cased.

Reply:
> but this change seems to be problematic for a workflows query when you have stopped workflows

Yeah, I think I need to change this back to `String`... The proto def has it defined as `string`, so unless you strip null values:
![image](https://github.com/user-attachments/assets/83aa7b12-d8e8-4002-85b1-d9b764023742)
(and setting a graphql default won't work, because the field exists on the PbWorkflow data)

Only other ways I can think of is:
- setting this field at the UIServer when it creates the store from the contact list (but what would you set it as?)..
- Or recording this in the Contact file, and for back compat set a default if not there.

Reply:
> Thoughts?

No thanks, I don't.

I'm not in favour of an unnecessary change here: I had a play with this with my UI branch which uses this data checked out and I had to add extra logic to make the runMode display correctly before and after your change.

Reply:
regressed to less consistent:
![image](https://github.com/user-attachments/assets/80805490-94da-43f0-8265-8e44b92abe3d)